### PR TITLE
[ENHANCEMENT] refactored config for contacts and accounts

### PR DESCRIPTION
**Pull request by michaelzangerle**
__Tasks:__
- [ ] gather feedback for my changes

__Informations:__

| Q             | A
| ------------- | ---
| Fixed tickets?      | no
| BC breaks?  | no
| Documentation PR?   | none

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@ CHANGELOG for Sulu
 ==================
 
 * dev-develop
+    * ENHANCEMENT #1523 [ContactBundle]   Refactored config to make is reusable by other bundles
```
Comment (turbo-ele) [inline CHANGELOG.md]: typo

File: src/Sulu/Bundle/ContactBundle/Resources/public/js/main.js [hunk 2]
```diff
@@ -89,21 +89,30 @@ define(['config', 'aura_extensions/iban'], function(Config, IbanExtension) {
...
+                {title: 'contact.accounts.title', link: 'contacts/accounts'}
+            ]);
+
+            Config.set('sulucontact.breadcrumb.contact', [
+                {title: 'navigation.contacts'},
+                {title: 'contact.contacts.title', link: 'contacts/contacts'}
+            ]);
+
+            // route to list
+            Config.set('sulucontact.routeToList.contact','contacts/contacts');
```
Comment (turbo-ele) [inline src/Sulu/Bundle/ContactBundle/Resources/public/js/main.js]: personally i'd prefer sulucontact.route.contact.list
or is ``routeToList`` used as identifier more often?

Comment (michaelzangerle) [inline src/Sulu/Bundle/ContactBundle/Resources/public/js/main.js]: it is already used for the filters

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,8 @@ CHANGELOG for Sulu
     * FEATURE     #1522 [SecurityBundle]  Created OrderByTrait for Repositories to sort by given array data
                                           `[field.name => order]` 
     * ENHANCEMENT #1522 [SecurityBundle]  Added sorting option for findUserByAccount in UserRepository 
+    * ENHANCEMENT #1523 [ContactBundle]   Refactored config to make is reusable by other bundles
```
Comment (turbo-ele) [inline CHANGELOG.md]: remove

**Comment by turbo-ele** [comment]
else ack

**Pull request merged**